Pinetree.Client/wwwroot/js/Markdown.js

```
@@ -0,0 +1,48 @@
+    const tooltipTriggerList = Array.from(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
+    tooltipTriggerList.forEach(tooltipTriggerEl => {
+        new window.bootstrap.Tooltip(tooltipTriggerEl);
+    });
+}
+export function setupLinkInterceptor(container, dotNetRef) {
+    container.addEventListener('click', (e) => {
+        const target = e.target;
+        if (target.tagName === 'A') {
+            const linkElement = target;
+            const href = linkElement.getAttribute('href');
+            if (href && href.startsWith('//')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)